docs/ref/models/instances.txt

```diff
@@ -637,10 +637,11 @@ with :func:`~django.utils.encoding.python_2_unicode_compatible` as shown above.
 
 .. method:: Model.__eq__()
 
-The equality method is defined such that instances with the same primary
-key value and the same concrete class are considered equal. For proxy
-models, concrete class is defined as the model's first non-proxy parent;
-for all other models it is simply the model's class.
+The equality method is defined such that saved instances with the same primary
```

Comment:
I think the proposed wording is slightly inaccurate because instances don't have to be saved to have a primary key. For example:
```
>>> MyModel(pk=1) == MyModel(pk=1)
True
```